### DISCUSSION TITLE
Help! Hasura load is way too heavy on our GCP SQL Instance

### DISCUSSION THREAD

**Discussion by Maxhodges** [hasura/graphql-engine#8973] [Question]
Hi we recently put Hasura into production for a new app, and it's killing our GCP SQL instance. It's using a lot of resources for such a small amount of transactions. This is not sustainable. We have another app on a much smaller SQL configuration and it handles a much larger amount of users and transactions. But this Hasura app users like 4-5X more CPU and connections.

Any tips on where to start troubleshooting? What are some of the first things we should look at? I keep adding more CPUs whenever it hits 100%, but this is not sustainable. What we can do to reduce the load on our database?

**Comment by meetzaveri** [comment]
Hey @Maxhodges 

We would encourage you to create a support ticket for these type of issues. You can visit Hasura cloud dashboard and can file a support ticket from there.

**Comment by dsandip** [comment]
@harishnair96
